articles/software-self-service.md

```diff
@@ -0,0 +1,102 @@
+# Software self-service
+
+![Software self-service](../website/assets/images/articles/software-self-service-1600x900@2x.png)
+
+Fleet’s self-service software feature empowers end users by allowing them to independently install approved software packages from a curated list through the Fleet Desktop “My device” page. This not only reduces the administrative burden on IT teams but also enhances user productivity and satisfaction. In this guide, we will walk you through the process of uploading, editing, and managing self-service software packages in Fleet, enabling seamless software distribution and management.
+
+
+## Prerequisites
+
```

Comment:
I think here and elsewhere some formatting issues with extra newlines?

Reply:
Good catch. My docs to md converter loves the newlines.